scripts/zones/Windurst_Waters_[S]/npcs/Door_Acolyte_hostel_up.lua

```diff
@@ -0,0 +1,33 @@
+-----------------------------------
+-- Area: Windurst Waters (S)
+--  NPC: Door Acolyte hostel
+-----------------------------------
+local ID = require("scripts/zones/Windurst_Waters_[S]/IDs")
+require("scripts/globals/keyitems")
+require("scripts/globals/missions")
+require("scripts/globals/settings")
+require("scripts/globals/quests")
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end
+
+function onTrigger(player,npc)
+    local sayItWithAHandbag = player:getQuestStatus(CRYSTAL_WAR, dsp.quest.id.crystalWar.SAY_IT_WITH_A_HANDBAG)
+    local sayItWithAHandbagCS = player:getVar("sayItWithAHandbagCS")
+    if (sayItWithAHandbag == QUEST_ACCEPTED and sayItWithAHandbagCS == 0) then
+        player:startEvent(171)
+    end
+end
+
+function onEventUpdate(player,csid,option)
+end
+
+function onEventFinish(player,csid,option)
+    if (csid == 171 and option == 1) then
+        player:setVar("sayItWithAHandbagCS", 1)
+    end
+    if (csid == 171 and option == 0) then
```

Comment:
`elseif`, or combine with an outer csid check:
```
if csid == 171 then
    if option == 1 then
        player:setVar("sayItWithAHandbagCS", 1)
    else
        -- bug: quest music
    end
end
```

Although is checking for option 0 even required? You're not doing anything with it; is it supposed to _do_ something and you're just noting the bug? If choosing option 0 is supposed to have zero effect on anything, you can just ignore it.

Reply:
@ibm2431 Well the issue here is that there' 2 pop up message for choices.
The first one if you choose Yes, it will lead to the second one, which if you choose No then the bug will show up.
If you choose No from the beginning then the CS will end without having that bug with the quest music still ongoing after the CS got declined. This is why I put it on 2 different if statements

Reply:
combined with outer csid check. Thank you 😸